launcher_go/v2/cli_runtime_test.go

```diff
@@ -0,0 +1,204 @@
+			BeforeEach(func() {
+				//response should be non-empty, indicating a running container
+				response := []byte{123}
+				CmdOutputResponse = response
+			})
+
+			It("should not run start commands", func() {
+				runner := ddocker.StartCmd{Config: "test"}
+				runner.Run(cli, &ctx)
+				checkStartCmdWhenStarted()
+			})
+
+			It("should run stop commands", func() {
+				runner := ddocker.StopCmd{Config: "test"}
+				runner.Run(cli, &ctx)
+				checkStopCmd()
+			})
+
+			It("should keep running during commits, and be post-deploy migration aware when using a web only container", func() {
+				runner := ddocker.RebuildCmd{Config: "web_only"}
+				runner.Run(cli, &ctx)
+
+				//initial build
+				cmd := GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker build"))
+
+				//migrate, skipping post deployment migrations
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker run"))
+				Expect(cmd.String()).To(ContainSubstring("--tags=db,migrate"))
+				Expect(cmd.String()).To(ContainSubstring("--env SKIP_POST_DEPLOYMENT_MIGRATIONS=1"))
+
+				// precompile
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker run"))
+				Expect(cmd.String()).To(ContainSubstring("--tags=db,precompile"))
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker commit"))
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker rm"))
+
+				// destroying
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker ps --all --quiet --filter name=web_only"))
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker stop --time 600 web_only"))
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker rm"))
+
+				// starting container --run command won't run because
+				// tests already believe we're running
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker ps --quiet"))
+
+				// run post-deploy migrations
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker run"))
+				Expect(cmd.String()).To(ContainSubstring("--tags=db,migrate"))
+				Expect(len(RanCmds)).To(Equal(0))
+			})
+
+			It("should stop with standalone", func() {
+				runner := ddocker.RebuildCmd{Config: "standalone"}
+
+				runner.Run(cli, &ctx)
+
+				//initial build
+				cmd := GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker build"))
+				cmd = GetLastCommand()
+
+				// stop
+				Expect(cmd.String()).To(ContainSubstring("docker ps --all --quiet --filter name=standalone"))
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker stop"))
+
+				// run migrate
+				cmd = GetLastCommand()
+				Expect(cmd.String()).To(ContainSubstring("docker run"))
+				Expect(cmd.String()).To(ContainSubstring("--tags=db,migrate"))
+				Expect(cmd.String()).ToNot(ContainSubstring("--env SKIP_POST_DEPLOYMENT_MIGRATIONS=1"))
```

Comment:
It seems like we are skipping post deploy migrations here but we don't seem to be running the migrations after.

Reply:
This test is to check standalone bootstraps, which does not run skip post deploy migrations, so no need to run migrations at the end.

Maybe easily missed, but the expects is `ToNot` rather than `To` here

Reply:
> Maybe easily missed, but the expects is ToNot rather than To here

Ahh good catch 👍